Week2/homework/map_filter.js

```diff
@@ -1,5 +1,9 @@
 'use strict';
 
+//2.1
+
 const numbers = [1, 2, 3, 4];
 
-// Add your code here
+const doubledOnlyOddnumbers = numbers.filter(number => number % 2 !== 0).map(number => number * 2)
```

Comment:
Excellent! As a minor point, many developers prefer to format a statement like this as:

```js
const doubledOnlyOddnumbers = numbers
  .filter(number => number % 2 !== 0)
  .map(number => number * 2);
```